waspc/data/Generator/templates/react-app/src/auth/forms/Signup.jsx

```diff
@@ -1,53 +1,19 @@
 {{={= =}=}}
 import React, { useState } from 'react'
-import { useHistory } from 'react-router-dom'
 
-import signup from '../signup.js'
-import login from '../login.js'
-import { errorMessage } from '../../utils.js'
+import Auth from './Auth'
 
-const SignupForm = () => {
-  const history = useHistory()
-
-  const [usernameFieldVal, setUsernameFieldVal] = useState('')
-  const [passwordFieldVal, setPasswordFieldVal] = useState('')
-
-  const handleSignup = async (event) => {
-    event.preventDefault()
-    try {
-      await signup({ username: usernameFieldVal, password: passwordFieldVal })
-      await login (usernameFieldVal, passwordFieldVal)
-
-      setUsernameFieldVal('')
-      setPasswordFieldVal('')
-
-      // Redirect to configured page, defaults to /.
-      history.push('{= onAuthSucceededRedirectTo =}')
-    } catch (err) {
-      console.log(err)
-      window.alert(errorMessage(err))
-    }
-  }
-  
+const SignupForm = ({ appearance, logo, socialLayout }) => {
```

Comment:
Since Login and Signup forms are now so minimal, maybe we could just directly expose Auth?

Reply:
I think we should probably do that, but keep these files if we want to help with backwards compatibility?

People wouldn't need two routes, they could change the `isLogin` dynamically on a single auth page.

Reply:
I agree! I'll reflect that in the docs then. I think we can even ditch the old forms? They were pretty ugly so I doubt anybody was using them :D